"Incomplete URL substring sanitization","Security checks on the substrings of an unparsed URL are often vulnerable to bypassing.","warning","The string [[""https://sentry.io""|""relative:///tests/sentry/integrations/jira/test_csp.py:42:20:42:38""]] may be at an arbitrary position in the sanitized URL.","/tests/sentry/integrations/jira/test_csp.py","42","20","42","58"
"Incomplete URL substring sanitization","Security checks on the substrings of an unparsed URL are often vulnerable to bypassing.","warning","The string [[""example.org""|""relative:///tests/sentry/integrations/gitlab/test_webhook.py:195:20:195:32""]] may be at an arbitrary position in the sanitized URL.","/tests/sentry/integrations/gitlab/test_webhook.py","195","20","195","48"
"Overly permissive regular expression range","Overly permissive regular expression ranges match a wider range of characters than intended. This may allow an attacker to bypass a filter or sanitizer.","warning","Suspicious character range that overlaps with \w in the same character class, and is equivalent to [+,\-.\/0-9:;<=>?@A-Z\[\\\]^_].","/src/social_auth/models.py","21","43","21","45"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/project_docs_platform.py:41:46:41:53""]].","/src/sentry/utils/integrationdocs.py","98","19","98","26"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/debug/debug_performance_issue.py:16:28:16:38""]].
This path depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/debug/mail.py:412:11:412:17""]].","/src/sentry/utils/samples.py","157","31","157","39"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/debug/debug_performance_issue.py:16:28:16:38""]].
This path depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/debug/mail.py:412:11:412:17""]].","/src/sentry/utils/samples.py","160","31","160","39"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/debug/debug_performance_issue.py:16:28:16:38""]].
This path depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/debug/mail.py:412:11:412:17""]].","/src/sentry/utils/samples.py","172","19","172","27"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/debug_files.py:227:20:227:26""]].","/src/sentry/utils/zip.py","61","29","61","53"
"Uncontrolled data used in path expression","Accessing paths influenced by users can allow an attacker to access unexpected resources.","error","This path depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/debug_files.py:227:20:227:26""]].","/src/sentry/utils/zip.py","64","23","64","30"
"Bad HTML filtering regexp","Matching HTML tags using regular expressions is hard to do right, and can easily lead to security issues.","warning","This regular expression does not match upper case <SCRIPT> tags.","/src/sentry/templatetags/sentry_assets.py","108","31","108","63"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/codeowners/__init__.py:87:35:87:35""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/codeowners/__init__.py","88","47","88","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/accept_project_transfer.py:58:34:58:34""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/accept_project_transfer.py","59","29","59","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/accept_project_transfer.py:86:34:86:34""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/accept_project_transfer.py","87","29","87","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/models/actor.py:182:30:182:30""]] flows to this location and may be exposed to an external user.","/src/sentry/models/actor.py","183","47","183","88"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/incidents/serializers/alert_rule.py:141:38:141:38""]] flows to this location and may be exposed to an external user.","/src/sentry/incidents/serializers/alert_rule.py","142","47","142","68"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/incidents/serializers/alert_rule.py:282:48:282:48""]] flows to this location and may be exposed to an external user.","/src/sentry/incidents/serializers/alert_rule.py","283","47","283","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/incidents/serializers/alert_rule.py:303:52:303:52""]] flows to this location and may be exposed to an external user.","/src/sentry/incidents/serializers/alert_rule.py","304","47","304","77"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/integrations/slack/webhooks/action.py:503:59:503:61""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/integrations/organization_integrations/details.py:120:36:120:36""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/event_reprocessable.py:62:35:62:35""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_code_mapping_codeowners.py:46:28:46:28""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_integration_repos.py:47:60:47:60""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_integration_serverless_functions.py:37:36:37:36""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_integration_serverless_functions.py:69:36:69:36""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_user_reports.py:105:28:105:28""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/integrations/vercel/webhook.py:406:48:406:48""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/integrations/vercel/webhook.py:425:48:425:48""]] flows to this location and may be exposed to an external user.","/src/sentry/api/base.py","390","25","390","31"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/base.py:402:30:402:30""]] flows to this location and may be exposed to an external user.","/src/sentry/api/base.py","403","37","403","42"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/base.py:408:30:408:30""]] flows to this location and may be exposed to an external user.","/src/sentry/api/base.py","409","37","409","42"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/base.py:448:38:448:38""]] flows to this location and may be exposed to an external user.","/src/sentry/api/base.py","449","37","449","42"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/check_am2_compatibility.py:38:29:38:29""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/check_am2_compatibility.py","42","17","46","17"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/chunk.py:160:27:160:29""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/chunk.py","162","29","162","47"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/data_export/endpoints/data_export.py:75:37:75:37""]] flows to this location and may be exposed to an external user.","/src/sentry/data_export/endpoints/data_export.py","77","51","77","56"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/data_export/endpoints/data_export.py:104:42:104:44""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/data_export/endpoints/data_export.py:104:42:104:44""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/data_export/endpoints/data_export.py:104:42:104:44""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/data_export/endpoints/data_export.py:104:42:104:44""]] flows to this location and may be exposed to an external user.","/src/sentry/data_export/endpoints/data_export.py","105","51","105","58"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/data_export/endpoints/data_export.py:127:44:127:48""]] flows to this location and may be exposed to an external user.","/src/sentry/data_export/endpoints/data_export.py","128","29","128","33"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/data_export/endpoints/data_export.py:166:35:166:35""]] flows to this location and may be exposed to an external user.","/src/sentry/data_export/endpoints/data_export.py","171","29","171","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/datascrubbing.py:144:26:144:26""]] flows to this location and may be exposed to an external user.","/src/sentry/datascrubbing.py","145","43","145","43"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/debug_files.py:314:46:314:46""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/debug_files.py","315","29","315","61"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/helpers/group_index/delete.py:117:35:117:37""]] flows to this location and may be exposed to an external user.","/src/sentry/api/helpers/group_index/delete.py","118","29","118","48"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_member/requests/invite/details.py:31:59:31:61""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_member/requests/invite/details.py","32","47","32","54"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/discover/endpoints/discover_key_transactions.py:146:33:146:35""]] flows to this location and may be exposed to an external user.","/src/sentry/discover/endpoints/discover_key_transactions.py","147","29","147","36"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_events.py:57:32:57:34""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/group_events.py","58","29","58","48"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_events.py:64:33:64:33""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/group_events.py","65","37","65","42"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_events.py:69:36:69:38""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/group_events.py","70","37","70","44"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_events.py:107:42:107:42""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/group_events.py","108","41","108","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_integration_details.py:119:36:119:36""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_integration_details.py:119:36:119:36""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_integration_details.py:119:36:119:36""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_integration_details.py:119:36:119:36""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/group_integration_details.py","120","29","120","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_integration_details.py:159:36:159:36""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/group_integration_details.py","160","29","160","58"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_integration_details.py:191:36:191:36""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/group_integration_details.py","192","29","192","58"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/group_integration_details.py:243:36:243:36""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/group_integration_details.py","244","29","244","58"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/integrations/sentry_apps/internal_app_token/index.py:48:38:48:38""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/integrations/sentry_apps/internal_app_token/index.py","49","29","49","34"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/base.py:352:29:352:31""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/base.py:352:29:352:31""]] flows to this location and may be exposed to an external user.","/src/sentry/api/bases/integration.py","43","33","43","39"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/internal/mail.py:43:29:43:29""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/internal/mail.py","46","25","46","40"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/serializers/rest_framework/notification_action.py:180:33:180:33""]] flows to this location and may be exposed to an external user.","/src/sentry/api/serializers/rest_framework/notification_action.py","182","51","182","76"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/bases/organization.py:357:33:357:33""]] flows to this location and may be exposed to an external user.","/src/sentry/api/bases/organization.py","358","37","358","62"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/incidents/endpoints/organization_alert_rule_index.py:65:37:65:39""]] flows to this location and may be exposed to an external user.","/src/sentry/incidents/endpoints/organization_alert_rule_index.py","66","33","66","40"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_artifactbundle_assemble.py:64:46:64:46""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_artifactbundle_assemble.py","65","29","65","61"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/bases/organization_events.py:201:47:201:51""]] flows to this location and may be exposed to an external user.","/src/sentry/api/bases/organization_events.py","210","37","210","43"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/bases/organization_events.py:211:35:211:39""]] flows to this location and may be exposed to an external user.","/src/sentry/api/bases/organization_events.py","214","37","214","43"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/bases/organization_events.py:215:52:215:56""]] flows to this location and may be exposed to an external user.","/src/sentry/api/bases/organization_events.py","217","37","217","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/bases/organization_events.py:222:44:222:48""]] flows to this location and may be exposed to an external user.","/src/sentry/api/bases/organization_events.py","225","37","225","43"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/bases/organization_events.py:226:36:226:40""]] flows to this location and may be exposed to an external user.","/src/sentry/api/bases/organization_events.py","241","41","241","50"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_events.py:213:33:213:35""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_events.py","214","30","214","32"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_events_spans_histogram.py:32:30:32:30""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_events_spans_histogram.py","33","47","33","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_events_spans_performance.py:212:30:212:30""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_events_spans_performance.py","213","47","213","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_events_trends.py:449:38:449:42""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_events_trends.py","450","37","450","41"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_group_index.py:290:33:290:33""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_group_index.py","291","37","291","42"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_group_index.py:391:66:391:68""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_group_index.py","392","29","392","48"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_group_index_stats.py:61:33:61:33""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_group_index_stats.py","62","37","62","42"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/incidents/endpoints/organization_incident_index.py:95:37:95:39""]] flows to this location and may be exposed to an external user.","/src/sentry/incidents/endpoints/organization_incident_index.py","96","33","96","40"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_issues_count.py:63:33:63:33""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_issues_count.py","64","37","64","42"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_issues_count.py:99:70:99:72""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_issues_count.py","100","33","100","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_metrics.py:63:33:63:33""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_metrics.py","64","40","64","40"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_metrics.py:65:63:65:65""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_metrics.py","66","37","66","44"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_metrics.py:93:64:93:66""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_metrics.py","94","38","94","45"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_metrics.py:115:64:115:66""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_metrics.py","121","34","121","36"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_metrics.py:157:18:157:20""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_metrics.py","158","42","158","49"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_pinned_searches.py:24:30:24:30""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_pinned_searches.py","25","47","25","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_pinned_searches.py:60:30:60:30""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_pinned_searches.py","61","29","61","86"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_profiling_profiles.py:23:38:23:40""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_profiling_profiles.py","24","37","24","44"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_recent_searches.py:40:30:40:30""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_recent_searches.py","41","29","41","86"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_recent_searches.py:45:30:45:30""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_recent_searches.py","46","29","46","87"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_release_assemble.py:52:46:52:46""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_release_assemble.py","53","29","53","61"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_release_details.py:481:45:481:45""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_release_details.py","483","37","483","54"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_release_details.py:519:41:519:41""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_release_details.py","520","29","520","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_releases.py:293:42:293:42""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_releases.py","295","21","295","38"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_releases.py:547:49:547:49""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_releases.py:547:49:547:49""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_releases.py:547:49:547:49""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_releases.py:547:49:547:49""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_releases.py","549","41","549","58"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_releases.py:610:42:610:42""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_releases.py","612","21","612","38"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/replays/endpoints/organization_replay_count.py:62:52:62:52""]] flows to this location and may be exposed to an external user.","/src/sentry/replays/endpoints/organization_replay_count.py","63","29","63","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/replays/endpoints/organization_replay_index.py:56:42:56:42""]] flows to this location and may be exposed to an external user.","/src/sentry/replays/endpoints/organization_replay_index.py","57","34","57","39"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_searches.py:34:30:34:30""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_searches.py","35","29","35","86"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_sessions.py:96:61:96:65""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_sessions.py","97","37","97","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/organization_stats_v2.py:215:93:215:97""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/organization_stats_v2.py","216","37","216","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_details.py:215:39:215:39""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/project_details.py","216","47","216","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_details.py:265:41:265:41""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/project_details.py","266","47","266","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_details.py:293:47:293:47""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/project_details.py","294","47","294","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_group_index.py:143:35:143:37""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_group_index.py:143:35:143:37""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_group_index.py:143:35:143:37""]] flows to this location and may be exposed to an external user.
[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_group_index.py:143:35:143:37""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/project_group_index.py","144","29","144","48"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_profiling_profile.py:31:38:31:40""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/project_profiling_profile.py","32","37","32","44"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_release_details.py:166:41:166:41""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/project_release_details.py","167","29","167","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/project_release_stats.py:61:38:61:38""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/project_release_stats.py","62","29","62","46"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/replays/endpoints/project_replay_clicks_index.py:66:42:66:42""]] flows to this location and may be exposed to an external user.","/src/sentry/replays/endpoints/project_replay_clicks_index.py","67","34","67","39"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/relay/register_challenge.py:77:29:77:31""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/relay/register_challenge.py","79","17","79","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/relay/register_response.py:62:29:62:31""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/relay/register_response.py","64","17","64","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/plugins/providers/repository.py:64:31:64:31""]] flows to this location and may be exposed to an external user.","/src/sentry/plugins/providers/repository.py","66","29","66","59"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/integrations/gitlab/search.py:43:32:43:32""]] flows to this location and may be exposed to an external user.","/src/sentry/integrations/gitlab/search.py","44","33","44","50"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/integrations/gitlab/search.py:59:32:59:32""]] flows to this location and may be exposed to an external user.","/src/sentry/integrations/gitlab/search.py","60","33","60","50"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/integrations/jira_server/search.py:45:40:45:40""]] flows to this location and may be exposed to an external user.","/src/sentry/integrations/jira_server/search.py","46","33","46","50"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/integrations/jira/endpoints/search.py:49:40:49:40""]] flows to this location and may be exposed to an external user.","/src/sentry/integrations/jira/endpoints/search.py","50","33","50","50"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/discover/endpoints/serializers.py:81:33:81:33""]] flows to this location and may be exposed to an external user.","/src/sentry/discover/endpoints/serializers.py","82","47","82","52"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/discover/endpoints/serializers.py:236:61:236:63""]] flows to this location and may be exposed to an external user.","/src/sentry/discover/endpoints/serializers.py","237","51","237","85"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/endpoints/system_options.py:113:51:113:51""]] flows to this location and may be exposed to an external user.","/src/sentry/api/endpoints/system_options.py","118","21","121","21"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/api/helpers/group_index/update.py:239:35:239:37""]] flows to this location and may be exposed to an external user.","/src/sentry/api/helpers/group_index/update.py","240","29","240","48"
"Information exposure through an exception","Leaking information about an exception, such as messages and stack traces, to an external user can expose implementation details that are useful to an attacker for developing a subsequent exploit.","error","[[""Stack trace information""|""relative:///src/sentry/integrations/msteams/webhook.py:117:25:117:27""]] flows to this location and may be exposed to an external user.","/src/sentry/integrations/msteams/webhook.py","119","36","119","71"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/auth_login.py:310:19:310:25""]].
Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/auth_login.py:338:20:338:26""]].
Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/oauth_authorize.py:74:19:74:25""]].
Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/oauth_authorize.py:210:20:210:26""]].","/src/sentry/web/frontend/auth_login.py","151","45","151","47"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/auth_login.py:338:20:338:26""]].
Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/oauth_authorize.py:210:20:210:26""]].","/src/sentry/web/frontend/auth_login.py","350","41","350","48"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/auth_login.py:310:19:310:25""]].
Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/auth_logout.py:23:20:23:26""]].
Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/oauth_authorize.py:74:19:74:25""]].","/src/sentry/web/frontend/base.py","408","36","408","38"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/web/frontend/debug/mail.py:315:20:315:26""]].","/src/sentry/web/frontend/debug/mail.py","325","25","325","36"
"URL redirection from remote source","URL redirection based on unvalidated user input may cause redirection to malicious web sites.","error","Untrusted URL redirection depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/project_profiling_profile.py:213:49:213:58""]].","/src/sentry/api/endpoints/project_profiling_profile.py","236","37","236","48"
"Polynomial regular expression used on uncontrolled data","A regular expression that can require polynomial time to match may be vulnerable to denial-of-service attacks.","warning","This [[""regular expression""|""relative:///src/sentry/utils/email/address.py:18:31:18:35""]] that depends on a [[""user-provided value""|""relative:///src/sentry/integrations/vsts/webhooks.py:27:20:27:26""]] may run slow on strings starting with '<' and with many repetitions of '<='.","/src/sentry/utils/email/address.py","65","35","65","39"
"Polynomial regular expression used on uncontrolled data","A regular expression that can require polynomial time to match may be vulnerable to denial-of-service attacks.","warning","This [[""regular expression""|""relative:///src/sentry/api/endpoints/chunk.py:24:95:24:96""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/chunk.py:50:19:50:25""]] may run slow on strings starting with 'sentry-cli/0.0.0' and with many repetitions of '0'.","/src/sentry/api/endpoints/chunk.py","62","56","62","65"
"Polynomial regular expression used on uncontrolled data","A regular expression that can require polynomial time to match may be vulnerable to denial-of-service attacks.","warning","This [[""regular expression""|""relative:///src/sentry/api/endpoints/debug_files.py:45:37:45:39""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/debug_files.py:402:19:402:25""]] may run slow on strings with many repetitions of ' '.
This [[""regular expression""|""relative:///src/sentry/api/endpoints/debug_files.py:45:43:45:47""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/debug_files.py:402:19:402:25""]] may run slow on strings starting with ' (' and with many repetitions of ' (('.","/src/sentry/api/endpoints/debug_files.py","428","50","428","54"
"Polynomial regular expression used on uncontrolled data","A regular expression that can require polynomial time to match may be vulnerable to denial-of-service attacks.","warning","This [[""regular expression""|""relative:///src/sentry/api/endpoints/organization_dashboards.py:18:30:18:31""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/organization_dashboards.py:128:20:128:26""]] may run slow on strings with many repetitions of 'a'.","/src/sentry/api/endpoints/organization_dashboards.py","164","51","164","55"
"Polynomial regular expression used on uncontrolled data","A regular expression that can require polynomial time to match may be vulnerable to denial-of-service attacks.","warning","This [[""regular expression""|""relative:///src/sentry/api/endpoints/organization_releases.py:59:37:59:39""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/organization_release_details.py:275:19:275:25""]] may run slow on strings with many repetitions of ' '.
This [[""regular expression""|""relative:///src/sentry/api/endpoints/organization_releases.py:59:43:59:47""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/organization_release_details.py:275:19:275:25""]] may run slow on strings starting with ' (' and with many repetitions of ' (('.","/src/sentry/api/endpoints/organization_release_details.py","63","46","63","50"
"Polynomial regular expression used on uncontrolled data","A regular expression that can require polynomial time to match may be vulnerable to denial-of-service attacks.","warning","This [[""regular expression""|""relative:///src/sentry/api/endpoints/organization_releases.py:59:37:59:39""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/organization_releases.py:236:19:236:25""]] may run slow on strings with many repetitions of ' '.
This [[""regular expression""|""relative:///src/sentry/api/endpoints/organization_releases.py:59:43:59:47""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/organization_releases.py:236:19:236:25""]] may run slow on strings starting with ' (' and with many repetitions of ' (('.
This [[""regular expression""|""relative:///src/sentry/api/endpoints/organization_releases.py:59:37:59:39""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/organization_releases.py:578:19:578:25""]] may run slow on strings with many repetitions of ' '.
This [[""regular expression""|""relative:///src/sentry/api/endpoints/organization_releases.py:59:43:59:47""]] that depends on a [[""user-provided value""|""relative:///src/sentry/api/endpoints/organization_releases.py:578:19:578:25""]] may run slow on strings starting with ' (' and with many repetitions of ' (('.","/src/sentry/api/endpoints/organization_releases.py","83","50","83","54"
"Polynomial regular expression used on uncontrolled data","A regular expression that can require polynomial time to match may be vulnerable to denial-of-service attacks.","warning","This [[""regular expression""|""relative:///src/sentry/scim/endpoints/teams.py:452:43:452:45""]] that depends on a [[""user-provided value""|""relative:///src/sentry/scim/endpoints/teams.py:307:21:307:27""]] may run slow on strings starting with '[' and with many repetitions of '[a'.","/src/sentry/scim/endpoints/teams.py","452","52","452","68"
